src/components/styles/bookItem.module.css

```diff
@@ -14,18 +14,19 @@
   transform: scale(1.02) !important;
 }
 
-.author {
-  font-size: 1rem !important;
-}
-
 .cardImg {
   height: 240px;
   object-fit: contain;
```

Comment:
![obraz](https://user-images.githubusercontent.com/110868938/231233633-0b3d754b-4462-4786-8665-b7c7a80f4158.png)

Zobacz czy nie bedzie to lepiej wygladalo jak zmienisz object-fit na fill (ewentualnie cover ale wtedy przytnie)